src/Compilers/VisualBasic/Test/Syntax/Parser/ParseErrorTests.vb

```diff
@@ -857,8 +859,8 @@ End Module
                 End Sub
             End Module
                 </file>
-        </compilation>).VerifyDiagnostics(Diagnostic(ERRID.ERR_RequiredConstExpr, "New Integer(1, 1) {{1, 2}, {2, 3}}"),
-            Diagnostic(ERRID.ERR_OverloadWithDefault2, "foo").WithArguments("Public Sub foo([i As Integer(*,*)])", "Public Sub foo([i As Integer(*,*) = Nothing])"))
+        </compilation>).VerifyDiagnostics(Diagnostic(ERRID.ERR_RequiredConstExpr, "New Integer(1, 1) {{1, 2}, {2, 3}}").WithLocation(5, 58),
```

Comment:
>VerifyDiagnostics [](start = 24, length = 17)

What is the source of the behavior change?
Please convert this test to use AssertTheseDiagnostics helper.